### PR TITLE
Made it PWA optimized

### PR DESCRIPTION
**Pull request by chinmaym07**
Fixes issue #11 
Please review @iharshit009

### CODE REVIEW

**Comment by chinmaym07** [comment]
Hey @iharshit009 why does that greeting check failed .? I don't know.Please help.

**Comment by iharshit009** [comment]
> Hey @iharshit009 why does that greeting check failed .? I don't know. Please help.

Leave that, I was trying to solve the issue but could find out any way till now. I will look over that later. Actually, it's missing some resources to run checks.

**Comment by iharshit009** [comment]
Screenshot from 2020-12-11 14-40-40
I am getting these errors on my console, could you please check them out. Are these shown to you too when you open the app?

File: index.html [hunk 1]
```diff
@@ -27,6 +38,71 @@ <h1>Write your confession here</h1>
...
+
+
+            }
+        }
+
+        /**
+         * Register Service Worker
+         */
+        navigator.serviceWorker
+            .register('/service-worker.js', { scope: '/' })
```
Comment (iharshit009) [inline index.html:86]: Uncaught (in promise) TypeError: Failed to register a ServiceWorker: The URL protocol of the current origin ('null') is not supported.

Comment (chinmaym07) [reply index.html:86]: I think you need to deploy this to check for service worker.

**Comment by chinmaym07** [comment]
Peek 2020-12-11 14-48

**Comment by chinmaym07** [comment]
Now you can check .It's fixed on my local machine.

File: index.html (hunk 1 above)
Comment (iharshit009) [reply index.html:86]: Okay

**Review by iharshit009 — COMMENTED** [review]
Looks Good.
Let this PR remain open for a while, till I deploy the app. As it needs some minor tests.
Will it be okay?

As soon as the socket work is complete, I would deploy the app

**Comment by chinmaym07** [comment]
Yes . :+1: 😀

**Pull request merged**